src/Akka.Persistence.Sql.Linq2Db/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -199,13 +194,33 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                     .IsColumn().IsNullable(false)
                     .HasLength(64)
                     .IsPrimaryKey()
-                    .Member(r => r.JournalOrderingId).HasColumnName(tagColumns.OrderingId)
-                    .IsColumn().IsPrimaryKey();
+                    .Member(r => r.SequenceNumber).HasColumnName(tagColumns.SequenceNumber)
+                    .IsColumn().IsNullable(false)
+                    .Member(r => r.PersistenceId).HasColumnName(tagColumns.PersistenceId)
+                    .HasLength(255)
```

Comment:
Asking because I don't know the answer: where did the 255 character constraint come from for PersistentId?

Reply:
Its the default value for all `PersistenceId` column for all SQL extensions

Reply:
Figures, ok